### PR TITLE
[4.4] Move the Cypress Tests to ESM

### PR DESCRIPTION
**Pull request by LadySolveig**
Pull Request for Issue # .

### Summary of Changes

-  Move to ESM (no require())
-  use .mjs for the esm modules

Thank you @dgrammatiko - This is only the continuition of the PR #42939 backported to 4.4 so we can upmerge in 5.1 and 5.2

### Testing Instructions

Review and passing automated tests

### Actual result BEFORE applying this Pull Request

Tests pass

### Expected result AFTER applying this Pull Request

Tests pass

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [X] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [X] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by laoneo** [comment]
Thanks as I really like the change. I'm merging this as last one when the other system test pr's are merged, so we have no conflicts in them.

File: tests/System/plugins/index.mjs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+// const mail = require('./mail');
+// const fs = require('./fs');
+// const db = require('./db');
+import { getMails, clearEmails, startMailServer } from "./mail.mjs";
```
Comment (laoneo) [inline tests/System/plugins/index.mjs:1]: Can you remove the comments above?

Comment (LadySolveig) [reply tests/System/plugins/index.mjs:1]: Thanks!

**Comment by laoneo** [comment]
Thanks!

**Pull request merged**